pkg/cockroach/cockroach.go

```diff
@@ -45,13 +48,106 @@ type DB struct {
 	Pool *pgxpool.Pool
 }
 
+func parseIntOrDefault(port string, defaultPort int64) int64 {
+	p, err := strconv.ParseInt(port, 10, 16)
+	if err != nil {
+		p = defaultPort
+	}
+	return p
+}
+
+// connectParametersFromMap constructs a ConnectParameters instance from m.
+func connectParametersFromMap(m map[string]string) ConnectParameters {
+	return ConnectParameters{
```

Comment:
Usually in (Google) go, structs are passed as pointers (this does a copy). The style of this codebase might be different.